### PR TITLE
New package/version: z-------------.cpod.Pre-release version 1.0.1

### PR DESCRIPTION
**Pull request by DuckDuckStudio**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/198900)

### CODE REVIEW

File: manifests/s/sf-yuzifu/bcm_convertor/1.3.0/sf-yuzifu.bcm_convertor.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+PublisherSupportUrl: https://github.com/sf-yuzifu/bcm_convertor/issues
+Author: sf-yuzifu
+PackageName: bcm converter
+PackageUrl: https://github.com/sf-yuzifu/bcm_convertor/
+License: GPL-3.0
+LicenseUrl: https://github.com/sf-yuzifu/bcm_convertor/blob/HEAD/LICENSE
+Copyright: Copyright (C) 2023-present sf-yuzifu
+CopyrightUrl: https://github.com/sf-yuzifu/bcm_convertor/blob/master/LICENSE
+ShortDescription: convert your own kitten3/4 into standalone package which runs on Windows/Mac OS/Linux quickly
+Moniker: bcm_convertor
```
Comment (DuckDuckStudio) [inline manifests/s/sf-yuzifu/bcm_convertor/1.3.0/sf-yuzifu.bcm_convertor.locale.en-US.yaml:18]:
```suggestion
#Moniker: bcm_convertor
```

File: manifests/s/sf-yuzifu/bcm_convertor/1.3.0/sf-yuzifu.bcm_convertor.locale.zh-CN.yaml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+PackageName: 编程猫格式工厂
+PackageUrl: https://gitee.com/sf-yuzifu/bcm_convertor/
+LicenseUrl: https://gitee.com/sf-yuzifu/bcm_convertor/blob/tauri/LICENSE
+Copyright: Copyright (C) 2023-present 小鱼yuzifu
+CopyrightUrl: https://gitee.com/sf-yuzifu/bcm_convertor/blob/tauri/LICENSE
+ShortDescription: 将自己在kitten3/4上的作品在Windows/Mac OS/Linux上快速制作成独立应用程序！
+ReleaseNotes: |-
+  1. 支持图标自定义
+  2. 修复Windows下转换完成无法打开桌面问题
+  3. 把是个浏览器的事实掩盖（bushi）
```
Comment (DuckDuckStudio) [inline manifests/s/sf-yuzifu/bcm_convertor/1.3.0/sf-yuzifu.bcm_convertor.locale.zh-CN.yaml:20]:
```suggestion
  3. 把是个浏览器的事实掩盖（bushi）
ReleaseNotesUrl: https://github.com/sf-yuzifu/bcm_convertor/releases/tag/v1.3.0
```

File: manifests/s/sf-yuzifu/bcm_convertor/1.3.0/sf-yuzifu.bcm_convertor.locale.zh-CN.yaml [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+Publisher: 小鱼yuzifu
+PublisherUrl: https://gitee.com/sf-yuzifu/
+PublisherSupportUrl: https://gitee.com/sf-yuzifu/bcm_convertor/issues
+Author: 小鱼yuzifu
+PackageName: 编程猫格式工厂
+PackageUrl: https://gitee.com/sf-yuzifu/bcm_convertor/
+LicenseUrl: https://gitee.com/sf-yuzifu/bcm_convertor/blob/tauri/LICENSE
+Copyright: Copyright (C) 2023-present 小鱼yuzifu
+CopyrightUrl: https://gitee.com/sf-yuzifu/bcm_convertor/blob/tauri/LICENSE
+ShortDescription: 将自己在kitten3/4上的作品在Windows/Mac OS/Linux上快速制作成独立应用程序！
```
Comment (DuckDuckStudio) [inline manifests/s/sf-yuzifu/bcm_convertor/1.3.0/sf-yuzifu.bcm_convertor.locale.zh-CN.yaml:16]:
```suggestion
ShortDescription: 将自己在kitten3/4上的作品在Windows/Mac OS/Linux上快速制作成独立应用程序！
Tags:
- 编程猫
- 转换器
- 打包器
- 工具
```

File: manifests/s/sf-yuzifu/bcm_convertor/1.3.0/sf-yuzifu.bcm_convertor.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,18 @@
+# Created with komac v2.8.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: sf-yuzifu.bcm_convertor
+PackageVersion: 1.3.0
+InstallerLocale: zh-CN
+InstallerType: nullsoft
+UpgradeBehavior: install
+ReleaseDate: 2022-08-19
```
Comment (DuckDuckStudio) [inline manifests/s/sf-yuzifu/bcm_convertor/1.3.0/sf-yuzifu.bcm_convertor.installer.yaml:9]:
```suggestion
ReleaseDate: 2022-08-19
FileExtensions:
  - bcm
```

File: manifests/s/sf-yuzifu/bcm_convertor/1.3.0/sf-yuzifu.bcm_convertor.locale.zh-CN.yaml [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+Tags:
+- 编程猫
+- 转换器
+- 打包器
+- 工具
```
Comment (DuckDuckStudio) [inline manifests/s/sf-yuzifu/bcm_convertor/1.3.0/sf-yuzifu.bcm_convertor.locale.zh-CN.yaml:21]:
```suggestion
- 工具
Moniker: bcm_convertor
```

**Comment by stephengillie** [comment]
Verify:
image
image

**Review by stephengillie — APPROVED** [review]

**Pull request merged**